### PR TITLE
Export tables to CSV

### PR DESCRIPTION
**Pull request by xuanjane**
# Export tables to CSV
by @therealyingtong @dzldebby @xuanjane 

([IRRC-14](https://bootcamp-irrc.atlassian.net/browse/IRRC-14)) Export student and teacher information
([IRRC-123](https://bootcamp-irrc.atlassian.net/browse/IRRC-123)) [Matched table] Add checkboxes to enable multiple selection for export purpose
Branch is based on #112 

## Description

Exports table information in each page (Screening, Matched, Students, Teachers) to CSV format.

## Test Steps

1. `npm install xlsx` on the frontend to install the xlsx library
2. Go to each page (Screening, Matched, Students, Teachers)
3. Click the Download/Download All/Download Selected button
4. Check that the information is downloaded into a CSV file

### CODE REVIEW

**Review by nondescryptid — APPROVED** [review]
Tested, all buttons could generate a csv file! Random question, but how are the entries sorted when we download them? Is there a way to sort (e.g. ascending ID number) before downloading? 
Screenshot 2021-04-10 at 7 18 11 PM

**Comment by xuanjane** [comment]
> Tested, all buttons could generate a csv file! Random question, but how are the entries sorted when we download them? Is there a way to sort (e.g. ascending ID number) before downloading?
> Screenshot 2021-04-10 at 7 18 11 PM

Hi Tomoe, thanks for approving the PR!
We didn't list any sort criteria for the export. Reason as thinking that it is easy for the user to sort with excel once the csv is out :)

File: frontend/src/pages/Teachers.vue [hunk 1]
```diff
@@ -137,6 +144,24 @@ export default {
...
+          ? teacher.secondLanguage.NativeLanguage
+          : "";
+        return {
+          TeacherID: `${teacher.TeacherID}`,
+          FullName: `${teacher.FullName}`,
+          CreatedAt: `${teacher.created_at}`,
+          Status: `${teacher.status.Description}`,
+          PhoneNumber: `${teacher.PhoneNumber}`,
+          Email: `${teacher.Email}`,
+          Language1: `${teacher.nativeLanguage.NativeLanguage}`,
```
Comment (thchia) [inline frontend/src/pages/Teachers.vue]: Not a big deal but I don't think you need the string templates here.

```js
{
  TeacherID: teacher.TeacherID,
  FullName: teacher.FullName,
  // ...
}
```

**Review by thchia — APPROVED** [review]
Untested, just one comment but it's not a blocker. Whether you change it or not, consider the PR approved.

File: frontend/src/pages/Teachers.vue (hunk 1 above)
Comment (xuanjane) [reply frontend/src/pages/Teachers.vue]: Have removed the string templates. Thank you Thomas!

**Pull request merged**